mvc/models/model-binding.rst

```diff
@@ -1,8 +1,113 @@
-.. include:: /../common/stub-topic.txt
+Model Binding
+=============
 
-|stub-icon| Model Binding Request Data
-======================================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-.. include:: /../common/stub-notice.txt
+In this article
 
-.. _issue: https://github.com/aspnet/Docs/issues/114
+- `Introduction to model binding`_
+- `How model binding works`_
+- `Customize default model binding behavior with attributes`_
+- `Binding formatted data from the request body`_ 
+- `Resources`_
+
+Introduction to model binding
```

Comment:
Not sure how we can introduce the model binding system without mentioning `ModelState`.